merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/CreatePlanDatasetAction.java

```diff
@@ -25,6 +28,7 @@ public CreatePlanDatasetAction(final Connection connection) throws SQLException
 
   public PlanDatasetRecord apply(
       final long planId,
+      final Optional<SimulationDatasetId> associatedSimulationDatasetId,
```

Comment:
This name change proposal is associated with the one I made on the db column
```suggestion
      final Optional<SimulationDatasetId> simulationDatasetId,
```